panel/forms.py

```diff
@@ -24,8 +29,10 @@ class VacQuery(forms.Form):
     accepted = forms.ChoiceField(choices=(('all', ("All")),
                                           (False, ("Not Accepted")),
                                           (True, ("Accepted"))))
-    search_from = forms.DateField(initial=date.today())
-    search_to = forms.DateField(initial=date.today())
+    search_from = forms.DateField(initial=date.today(),
+                                  widget=DateInput(attrs={'class': 'datepicker'}))
```

Comment:
Isn't this doubled with `widgets` in meta class?

Reply:
Nope. Two different forms :)